docs/api-reference/next.config.js/url-imports.md

```diff
@@ -27,6 +27,12 @@ import { a, b, c } from 'https://example.com/modules/some/module.js'
 
 URL Imports can be used everywhere normal package imports can be used.
 
+## Security Model
+
+URL imports provide an opportunity for stronger security of sharing ES modules than what exists today.
+
+This feature is being designed with **security as the top priority**. To start, we added an experimental flag forcing you to explicitly allow the domains you accept URL imports from. Next, we're working to take this further by limiting URL imports to execute in the browser sandbox using the [Edge Runtime](/docs/api-reference/edge-runtime.md). This runtime is used by [Middleware](/docs/middleware.md) as well as [Next.js Live](https://vercel.com/live).
```

Comment:
Can we avoid starting a sentence with “Next”? How about “In the future” instead?